lib/compiler/asts.js

```diff
@@ -110,10 +110,8 @@ const asts = {
         return rule ? consumes(rule) : undefined;
       },
 
-      library_ref() {
-        // No way to know for external rules.
-        return false;
-      },
+      // No way to know for external rules.
+      library_ref: consumesFalse,
```

Comment:
Unrelated change to remove an uncovered line.  I'll do another coverage PR after this lands.